docs/geedocs/5.3.0/answer/3499964.html

```diff
@@ -23,8 +23,7 @@ <h2>Uninstalling</h2>
 <li>Stop the Google Earth Enterprise Server, if you have a previous version installed and it is running:
 <p><code>/etc/init.d/geserver stop</code></p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the script <code>uninstall_server.sh</code>.</li></ol></div>
+<li>Navigate to the location at which you initially cloned the GEE git repo or to the installation location (<code>/opt/google</code>).</li>
+<li>If you're uninstalling from the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>.  If you're uninstalling from the installation location, navigate to <code>install</code>.  Now run the script <code>uninstall_server.sh</code>.</li></ol></div>
 </ul><div class="footer"><p class="BackToTop"><a href="#top_of_file">Back to top</a> <hr /></p> <p class="copyright">&copy;2015 Google</p></div>
```

Comment:
At the page footer let's add `©2019 Open GEE Contributors` to the copyright notice be: `©2015 Google, ©2019 Open GEE Contributors)`

Reply:
done